cmd/imagedigestexporter/main.go

```diff
@@ -63,18 +62,20 @@ func main() {
 			logger.Fatalf("Unexpected error getting image digest for %s: %v", imageResource.Name, err)
 		}
 		// We need to write both the old Name/Digest style and the new Key/Value styles.
-		output = append(output, v1alpha1.PipelineResourceResult{
-			Name:   imageResource.Name,
-			Digest: digest.String(),
-		})
-
-		output = append(output, v1alpha1.PipelineResourceResult{
-			Key:   "digest",
-			Value: digest.String(),
-			ResourceRef: v1alpha1.PipelineResourceRef{
-				Name: imageResource.Name,
-			},
-		})
+		output = append(output,
+			v1alpha1.PipelineResourceResult{
+				Key:   "digest",
+				Value: digest.String(),
+				ResourceRef: &v1alpha1.PipelineResourceRef{
+					Name: imageResource.Name,
+				},
+			}, v1alpha1.PipelineResourceResult{
+				Key:   "digest",
+				Value: digest.String(),
+				ResourceRef: &v1alpha1.PipelineResourceRef{
+					Name: imageResource.Name,
+				},
+			})
```

Comment:
I am confused why we need this same digest string twice.  Is there a reason for that or mistype?